AfxHookSource2/ViewModel.cpp

```diff
@@ -166,29 +182,44 @@ void ViewModel_Console(advancedfx::ICommandArgs* args)
 					setLeftHanded
 				);
 				return;
-				
-			}
+			};
 			advancedfx::Message(
-				"%s set <OffsetX|*> <OffsetY|*> <OffsetZ|*> <FOV|*> <Hand|*> - Set viewmodel. Use * to indicate to not change.\n"
-				"Hand: 0 = Right Handed, 1 = Left Handed\n"
-				"Current value: OffsetX: %f, OffsetY: %f, OffsetZ: %f, FOV: %f, Hand: %i\n"
+				"%s set <OffsetX|*> <OffsetY|*> <OffsetZ|*> <FOV|*> <Hand|*>\n"
+				"Set viewmodel. Use * to indicate to not change. Hand: 0 = Right Handed, 1 = Left Handed\n"
+				"Current value: OffsetX: %.1f, OffsetY: %.1f, OffsetZ: %.1f, FOV: %.1f, Hand: %i\n"
+				"Example:\n"
+				"%s set 2 2.5 -2 68 0\n"
 				, args->ArgV(0)
 				, g_MirvViewmodel.m_OffsetX
 				, g_MirvViewmodel.m_OffsetY
 				, g_MirvViewmodel.m_OffsetZ
 				, g_MirvViewmodel.m_Fov
 				, g_MirvViewmodel.m_leftHanded
+				, args->ArgV(0)
 			);
-		}
+		} else 
+		if (0 ==_stricmp("reset", subcmd)) {
+			g_MirvViewmodel.reset();
+		};
 	} else {
 		advancedfx::Message("%s enabled <1|0> - Enables/Disables the viewmodel\n", args->ArgV(0));

```

Comment:
I think we usually format the binary option as 0|1? May need to swap the wording too.

Also see https://github.com/advancedfx/advancedfx/pull/956/files#diff-6c487434dd237a1710d77f708301cb8507a7143f3507b1f6744fbeb050dfca7aR159

Reply:
I see, will fix